HdInsightSDK/src/com/microsoft/azure/hdinsight/sdk/cluster/HDInsightClusterDetail.java

```diff
@@ -0,0 +1,115 @@
+package com.microsoft.azure.hdinsight.sdk.cluster;
+
+import com.google.gson.annotations.Expose;
+import com.microsoft.azure.hdinsight.sdk.common.HDIException;
+import com.microsoft.azure.hdinsight.sdk.storage.StorageAccount;
+import com.microsoft.azure.hdinsight.sdk.subscription.Subscription;
+
+import java.io.IOException;
+import java.io.NotActiveException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by ltian on 12/23/2015.
+ */
+public class HDInsightClusterDetail implements IClusterDetail {

```

Comment:
try to rename it , all the clusters are HDinsight cluster, maybe ManuallyAddedClusterdetail is ok